modules/dapps.js

```diff
@@ -81,21 +82,21 @@ class DApps {
 			transactionTypes.DAPP
 		] = library.logic.transaction.attachAssetType(
 			transactionTypes.DAPP,
-			new DApp(scope.db, scope.logger, scope.schema, scope.network)
+			new DApp(scope.logger, scope.schema, scope.network, scope.storage)
 		);
 
 		__private.assetTypes[
 			transactionTypes.IN_TRANSFER
 		] = library.logic.transaction.attachAssetType(
 			transactionTypes.IN_TRANSFER,
-			new InTransfer(scope.db, scope.schema)
+			new InTransfer(scope.db, scope.schema, scope.storage)
```

Comment:
Why is `InTransfer` still using `scope.db`?

Reply:
Good catch. Not using it but need to remove the parameter and check usages in the rest of the code.

Reply:
Updated. Resolving.